src/languages/mailmojo-nb_NO.po

```diff
@@ -13,154 +13,164 @@ msgstr ""
 msgid "Signup Button Text"
 msgstr "Påmeldingsknapp"
 
-#: mailmojo-widget.php:126
-msgid "Optional Tags"
-msgstr "Valgfrie merkelapper"
+#: templates/widget-admin.php:47
+msgid "Include name field"
+msgstr "Inkluder navnfelt"
 
-#: mailmojo-widget.php:127
-msgid "Tag Selection Label"
-msgstr "Overskrift for merkelapper"
+#: templates/widget-admin.php:52
+msgid "Selectable tags"
+msgstr "Merkelapper som kan velges"
 
-#: mailmojo-widget.php:128
-msgid "Tags (comma separated)"
-msgstr "Merkelapper (kommaseparert)"
+#: templates/widget-admin.php:53 templates/widget-admin.php:93
+msgid "optional"
+msgstr "valgfri"
 
-#: mailmojo-widget.php:169
-msgid "Name"
-msgstr "Navn"
+#: templates/widget-admin.php:57
+msgid "Tag selection label"
+msgstr "Overskrift for valgfri merkelapper"
 
-#: mailmojo-widget.php:202
-msgid "E-mail"
-msgstr "E-postadresse"
+#: templates/widget-admin.php:65
+msgid "Type"
+msgstr "Type"
 
-#~ msgid ""
-#~ "You need to enter your MailMojo account information on the MailMojo "
-#~ "settings page"
-#~ msgstr ""
-#~ "Du må fylle ut din MailMojo-kontoinformasjon under innstillinger for "
-#~ "MailMojo"
+#: templates/widget-admin.php:71
+msgid "Single choice"
+msgstr "Enkelvalg"
 
-#~ msgid "Click to add more"
-#~ msgstr "Legg til flere"
+#: templates/widget-admin.php:78
+msgid "Multiple choice"
+msgstr "Flervalg"
 
-#~ msgid "You need to enter your MailMojo account information on the"
-#~ msgstr "Du må fylle ut din MailMojo-kontoinformasjon under"
+#: templates/widget-admin.php:83 templates/widget-admin.php:98
+msgid "Tags (comma separated)"
+msgstr "Merkelapper (kommaseparert)"
 
-#~ msgid "MailMojo settings page"
-#~ msgstr "innstillinger for MailMojo"
+#: templates/widget-admin.php:92
+msgid "Fixed tag(s)"
+msgstr "Faste merkelapper"
 
-#~ msgid "Ta-da! You've successfully signed up. Thank you!"
-#~ msgstr "Ta-da! Du er nå meldt på nyhetsbrevet. Tusen takk!"
+#: templates/widget-admin.php:95
+msgid "These tags will automatically be added to the subscribed contact."
+msgstr ""
+"Disse merkelappene vil bli automatisk lagt til på den som melder seg på."
 
-#~ msgid "Notifications"
-#~ msgstr "Meldinger"
+#: templates/widget-page.php:18
+msgid "E-mail"
+msgstr "E-postadresse"
 
-#~ msgid "Success Message"
-#~ msgstr "Vellykket påmelding"
+#: templates/widget-page.php:28
+msgid "Name"
+msgstr "Navn"
 
-#~ msgid "You must provide an e-mail address."
-#~ msgstr "Du må fylle ut en e-postadresse."
+#. Plugin Name of the plugin/theme
+msgid "MailMojo Widget"
+msgstr ""
 
-#~ msgid "Invalid e-mail address."
-#~ msgstr "Ugyldig e-postadresse."
+#. Plugin URI of the plugin/theme
+msgid "http://github.com/eliksir/MailMojo-WP-Widget"
+msgstr ""
 
-#~ msgid "An unknown error occured."
-#~ msgstr "Det har oppstått en ukjent feil."
+#. Description of the plugin/theme
+msgid ""
+"Adds a signup widget for a MailMojo mailing list to your WordPress site."
+msgstr ""
+"Legger til påmeldingskjema for MailMojo epostliste på ditt WordPress "
```

Comment:
"Legger til påmeldingsskjema for nyhetsbrev fra MailMojo til ditt WordPress-nettsted."